notification/src/lambda.js

```diff
@@ -0,0 +1,30 @@
+const ctp = require('./utils/ctp')
+const handler = require('./handler/notification/notification.handler')
+const config = require('./config/config')()
+const logger = require('./utils/logger').getLogger()
+
+const setup = require('./config/init/ensure-interface-interaction-custom-type')
+const ctpClient = ctp.get(config)
+
+let init = false;
```

Comment:
In extension module, it's called `initialised`. Let's choose one name and use it so it's consistent.

Reply:
Agreed, have changed this.